src/Standard.Licensing/LicenseAttributes.cs

```diff
@@ -63,8 +63,10 @@ public virtual void Add(string key, string value)
         /// <param name="features">The dictionary of elements.</param>
         public virtual void AddAll(IDictionary<string, string> features)
         {
-            foreach (var feature in features)
+            foreach (KeyValuePair<string, string> feature in features)
+            {
                 Add(feature.Key, feature.Value);
+            }
```

Comment:
Change from var. This could potentially be cleaned up a little with a var and deconstruction